<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signupWindowClass</class>
 <widget class="QMainWindow" name="signupWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>signupWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="signupButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>360</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Signup</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="signupUsernameLine">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="signupLogginButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>410</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sigupPasswordLine">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>260</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>410</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Already have an account?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sigupPasswordRepeatLine">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>310</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>480</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1052</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
